src/internal/operations/helpers/helpers.go

```diff
@@ -1,4 +1,4 @@
-package operations
+package helpers
```

Comment:
Generic package names like `"helpers"` are generally considered a smell by the golang community, as they indicate that the code either doesn't have a distinct enough identity to warrant separate ownership or that the identity is misunderstood, and the result often turns into a sort of dumping ground for code that could often be refined towards an identity

In this case, I think it's the latter: that the identity is misunderstood.  We don't want a blob of "helper" funcs.  So far, this package only contains errors-oriented functionality, so that sounds like the best choice of names: `/operations/errors/errors.go`.

That said, I'm also dubious about the need to move these files out at all.  The only imports are pkg/ control, fault, and logger, none of which are going to create an import cycle.  Seems to me that you could keep this in the original package as `operations/errors.go`.  That would help reduce additional imports, additional subpackages, and reduce the number of packages with a common name (ie: errors).  Unless I'm missing something about the imports, is it possible to keep this in the original package?